src/interfaces/blog.interfaces.tsx

```diff
@@ -0,0 +1,23 @@
+import { IApiResponse } from './api-response.interface';
+
+export interface IBlog {
+	id: string;
+	title: string;
+	slug: string;
+	excerpt: string;
+	content: string;
+	author: string;
+	publishedDate: string;
+	isPublished: boolean;
+	tags: string;
+	createdDate: string;
+	modifiedDate: string;
+}
+
+export interface IBlogListApiResponse extends IApiResponse {
+	value: IBlog[];
+}
+
+export interface IFetchBlogApiResponse extends IApiResponse {
```

Comment:
These fields do not match https://github.com/Golf-Course-Project/course-service/blob/main/CourseService/Entities/Blog.cs

Here is the class

  public class Blog
  {
      [Key]
      public string Id { get; set; }
      
      public string Title { get; set; }
      
      public string PageName { get; set; }
      
      public string Description { get; set; }

      public string ShortDescription { get; set; }

      public string DefaultImagePath { get; set; }

      public bool IsActive { get; set; } = true;

      public bool IsDraft { get; set; } = false;

      public DateTime DateCreated { get; set; }
  }